pkg/3scale/amp/operator/deployment_config_reconciler.go

```diff
@@ -127,3 +128,46 @@ func NewCreateOnlyDCReconciler() *CreateOnlyDCReconciler {
 func (r *CreateOnlyDCReconciler) IsUpdateNeeded(desired, existing *appsv1.DeploymentConfig) bool {
 	return false
 }
+
+func DeploymentConfigReconcileDeploymentTriggerImageChangeParamsName(desired, existing *appsv1.DeploymentConfig, logger logr.Logger) bool {
+	update := false
+	findDeploymentTriggerOnImageChange := func(triggerPolicies []appsv1.DeploymentTriggerPolicy) (int, error) {
+		result := -1
+		for i := range triggerPolicies {
+			if triggerPolicies[i].Type == appsv1.DeploymentTriggerOnImageChange {
+				if result != -1 {
+					return -1, fmt.Errorf("found more than one imageChangeParams Deployment trigger policy")
```

Comment:
You can have more than one ImageChange deploymentconfig trigger at the same time but I'm going to assume this should not happen in our DeploymentConfigs

Reply:
already commented